src/main/java/Joey.java

```diff
@@ -0,0 +1,218 @@
+        return "[T]";
+    }
+}
+
+
+//class Deadline extends Task {
+//    private String by;
+//
+//    public Deadline(String description, String by) {
+//        super(description);
+//        this.by = by;
+//    }
+//
+//    public String getType() {
+//        return "[D]";
+//    }
+//
+//    public String toString() {
+//        return super.toString() + " (by: " + by + ")";
+//    }
+//}
+
+class Deadline extends Task {
+    private String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    public String getType() {
+        return "[D]";
+    }
+}
+
+
+//class Event extends Task {
+//    private String from;
+//    private String to;
+//
+//    public Event(String description, String from, String to) {
+//        super(description);
+//        this.from = from;
+//        this.to = to;
+//    }
+//
+//    public String getType() {
+//        return "[E]";
+//    }
+//
+//    public String toString() {
+//        return getType() + getStatusIcon() + " " + super.toString() + " (from: " + from + " to: " + to + ")";
+//    }
+//}
+
+class Event extends Task {
+    private String from;
+    private String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    public String getType() {
+        return "[E]";
+    }
+
+    public String toString() {
+        return super.toString() + " (from: " + from + " to: " + to + ")";
+    }
+}
+
+
+public class Joey {
+    public static void main(String[] args) {
+        System.out.println("Hello! I'm Joey");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        while (true) {
+            String userCommand = scanner.nextLine();
+            System.out.println("____________________________________________________________");
+
+            if (userCommand.equalsIgnoreCase("bye")) {
+                System.out.println(" bye bye, take care:)!");
+                System.out.println("____________________________________________________________");
+                break;
+            } else if (userCommand.equalsIgnoreCase("list")) {
+                System.out.println(" Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
+                    Task task = tasks[i];
+                    System.out.println(" " + (i + 1) + "." + task.getType() + "[" + (task.isDone() ? "X" : " ") + "] " + task.getDescription());
+                }
+                System.out.println("____________________________________________________________");
+            } else if (userCommand.startsWith("todo")) {
+                String description = userCommand.substring(5);
```

Comment:
Should 5 be turned into a variable to remove the magic number?
```suggestion
                int todoLength = "todo".length();
                String description = userCommand.substring(todoLength);
```